packages/mjml-button/src/index.js

```diff
@@ -83,6 +83,8 @@ class Button extends Component {
         lineHeight: mjAttribute('line-height'),
         textDecoration: mjAttribute('text-decoration'),
         textTransform: mjAttribute('text-transform'),
+        padding: defaultUnit(mjAttribute('inner-padding'), "px"),
```

Comment:
Well sadly, most of desktop outlook doesn't support padding on `a` element even with an `inline-block`